Gemfile

```diff
@@ -20,6 +20,9 @@ gem "hanami-webconsole", github: "hanami/webconsole", branch: "main"
 
 gem "hanami-devtools", github: "hanami/devtools", branch: "main"
 
+gem "dry-system", github: "alassek/dry-system", branch: 'provider-class-override'
```

Comment:
change this back once dry-rb/dry-system#275 is merged

Reply:
👍🏼 — let's get https://github.com/dry-rb/dry-system/pull/275 merged first, so we can switch this to `branch: "main"` before merging.

Reply:
Have adjusted this back to dry-rb/dry-system main 👍🏼